src/torchgems/train_spatial.py

```diff
@@ -22,6 +22,46 @@
 import torch.distributed as dist
 
 
+def isPowerTwo(num):

```

Comment:
put this in a utils file instead of redefining

Reply:
Added this in utils.py